src/constants/images.ts

```diff
@@ -4,4 +4,40 @@ export const SVGS = {
     url: '/svgs/profile.svg',
     alt: '프로필',
   },
+  profileDefault: {
+    url: '/svgs/profileDefault.svg',
+    alt: '기본 프로필',
+  },
+  profileActive: {
+    url: '/svgs/profileActive.svg',
+    alt: '활성화된 프로필',
+  },
+  arrowDefault: {
+    url: '/svgs/arrowDefault.svg',
+    alt: '기본 화살표',
+  },
+  arrowActive: {
+    url: '/svgs/arrowActive.svg',
+    alt: '활성화된 화살표',
```

Comment:
default / active 묶어서 하나의 아이콘 안에 넣으면 어떨까요?
추가로 arrow 종류가 다양해서 구분되게 명칭 변경해 주시면 좋을 것 같습니다!
arrow 종류: top/bottom/left/right 등등

```suggestion
  arrow: {
    default: {
      url: '/svgs/arrowDefault.svg',
      alt: '기본 화살표',
    },
    active: {
      url: '/svgs/arrowActive.svg',
      alt: '활성화된 화살표',
    },
  },
```

Reply:
```suggestion
  arrow: {
    bottom: {
      url: '/svgs/arrowBottom.svg',
      alt: '아래쪽 화살표',
    },
    top: {
      url: '/svgs/arrowTop.svg',
      alt: '위쪽 화살표',
    },
  },
```
수정했습니다